spec/models/order_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  subject { Order.new( product_name: "gears", product_count: 7, customer: FactoryBot.create(:customer))}
+  it "is valid with valid attributes" do
+    expect(subject).to be_valid
+  end
+  it "is not valid without a product_name" do
+    subject.product_name=nil
+    expect(subject).to_not be_valid
+  end
+  it "is not valid without a product_count" do
+    subject.product_count=nil
+    expect(subject).to_not be_valid
+  end
+  it "is not valid without a customer" do
+    subject.customer=nil
+    expect(subject).to_not be_valid
+  end
+  it "is not valid without a customer_id" do
+    subject.customer_id=nil
+    expect(subject).to_not be_valid
+  end
+  it "returns the correct customer_id" do
+    expect(subject.customer_id).to eq(1)
```

Comment:
Assuming only one customer in your test database, and that its `id` is 1 is a reasonable assumption to make, but it might be better to assign the customer to a variable, and then asserting `subject.customer_id` to be equal to `customer.id`